### PR TITLE
name covariant_class -> covariantClass + attribute

### PR DESCRIPTION
**Pull request by arienmalec**
renamed covariant_class to covariantClass + #align statements

### CODE REVIEW

File: Mathlib/Algebra/Order/Monoid.lean [hunk 1]
```diff
@@ -38,46 +38,49 @@ attribute [to_additive OrderedAddCommMonoid] OrderedCommMonoid
...
 
 -- TODO `to_additive` should copy this
-attribute [instance] Add.to_covariant_class_left
+attribute [instance] Add.to_covariantClass_left
 
 /- This is not an instance, to avoid creating a loop in the type-class system: in a
-`right_cancel_semigroup` with a `partial_order`, assuming `covariant_class M M (swap (*)) (<)`
-implies `covariant_class M M (swap (*)) (≤)`, see
+`right_cancel_semigroup` with a `partial_order`, assuming `covariantClass M M (swap (*)) (<)`
+implies `covariantClass M M (swap (*)) (≤)`, see
```
Comment (mcdoll) [inline Mathlib/Algebra/Order/Monoid.lean]:
```suggestion
`RightCancelSemigroup` with a `PartialOrder`, assuming `CovariantClass M M (swap (*)) (<)`
implies `CovariantClass M M (swap (*)) (≤)`, see
```
plus change the line below that accordingly.

File: Mathlib/Algebra/Order/Monoid.lean [hunk 2]
```diff
@@ -38,46 +38,49 @@ attribute [to_additive OrderedAddCommMonoid] OrderedCommMonoid
...
-attribute [instance] OrderedAddCommMonoid.to_covariant_class_right
+attribute [instance] OrderedAddCommMonoid.to_covariantClass_right
 
 /- This is not an instance, to avoid creating a loop in the type-class system: in a
-`left_cancel_semigroup` with a `partial_order`, assuming `covariant_class M M (*) (≤)` implies
-`covariant_class M M (*) (<)`, see `left_cancel_semigroup.covariant_mul_lt_of_covariant_mul_le`. -/
-@[to_additive Add.to_covariant_class_left]
-theorem Mul.to_covariant_class_left (M : Type _) [Mul M] [PartialOrder M]
+`left_cancel_semigroup` with a `partial_order`, assuming `covariantClass M M (*) (≤)` implies
+`covariantClass M M (*) (<)`, see `left_cancel_semigroup.covariant_mul_lt_of_covariant_mul_le`. -/
```
Comment (mcdoll) [inline Mathlib/Algebra/Order/Monoid.lean]:
```suggestion
`LeftCancelSemigroup` with a `PartialOrder`, assuming `CovariantClass M M (*) (≤)` implies
`CovariantClass M M (*) (<)`, see `LeftCancelSemigroup.covariant_mul_lt_of_covariant_mul_le`. -/
```

File: Mathlib/Algebra/Order/Monoid.lean [hunk 3]
```diff
@@ -38,46 +38,49 @@ attribute [to_additive OrderedAddCommMonoid] OrderedCommMonoid
...
 
 -- TODO `to_additive` should copy this
-attribute [instance] OrderedAddCommMonoid.to_covariant_class_left
+attribute [instance] OrderedAddCommMonoid.to_covariantClass_left
 
 /- This instance can be proven with `by apply_instance`.  However, `with_bot ℕ` does not
-pick up a `covariant_class M M (function.swap (*)) (≤)` instance without it (see PR #7940). -/
-@[to_additive OrderedAddCommMonoid.to_covariant_class_right]
-instance OrderedCommMonoid.to_covariant_class_right (M : Type _) [OrderedCommMonoid M] :
+pick up a `covariantClass M M (function.swap (*)) (≤)` instance without it (see PR #7940). -/
```
Comment (mcdoll) [inline Mathlib/Algebra/Order/Monoid.lean]:
```suggestion
pick up a `CovariantClass M M (function.swap (*)) (≤)` instance without it (see PR #7940). -/
```

**Review by mcdoll — COMMENTED** [review]
Minor corrections

**Comment by arienmalec** [comment]
Incorrect PR based on mistaken push to master

**Pull request closed without merging**